lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/DesktopEnvironment.java

```diff
@@ -0,0 +1,39 @@
+package org.enso.desktopenvironment;
+
+import org.enso.common.Platform;
+
+public final class DesktopEnvironment {
```

Comment:
To minimize conceptual surface, make this package private:
```suggestion
final class DesktopEnvironment {
```
and move the static into `Directories.getCurrent()` and `TrashBin.getCurrrent()` to mimic the API of `WorkingDirectory`